Frameworks/CoreFoundation/URL.subproj/CFURL.c

```diff
@@ -1323,8 +1323,17 @@ static Boolean _stringContainsCharacter(CFStringRef string, UniChar ch) {
     return false;
 }
 
+// WINOBJC : A function for creating percent escapes based on character sets instead of strings has been added. This is used for stringByAddingPercentEscapesUsingEncoding which operates on a valid set of 
```

Comment:
Please break these into shorter lines - I know clang isn't enabled on this file, but it's good practice nonetheless.